File: Php/core/HomegamePageModel.php
namespace core{

	use entities\Cashgame;
	use entities\Homegame;
	use app\Homegame\HomegameNavigationModel;
	use Domain\Classes\User;

	class HomegamePageModel extends PageModel{

		public $homegameNavigationModel;

		public function __construct(User $user = null,
									Homegame $homegame = null,
									Cashgame $runningGame = null){
			parent::__construct($user);
			if($homegame != null){
				$this->homegameNavigationModel = new HomegameNavigationModel($homegame, $runningGame);
			}
		}

	}

}

File: Web/Models/PageBaseModels/HomegamePageModel.cs
using Core.Classes;
using Web.Models.NavigationModels;

namespace Web.Models.PageBaseModels{

	public class HomegamePageModel : PageModel{

	    public HomegamePageModel(User user, Homegame homegame, Cashgame runningGame) : base(user)
	    {
	        if(homegame != null){
			    HomegameNavModel = new HomegameNavigationModel(homegame, runningGame);
			}
	    }

	}

}